app/src/components/v-form/form-field.vue

```diff
@@ -174,15 +177,15 @@ export default defineComponent({
 			isEdited,
 		};
 
-		function emitValue(value: any) {
+		function emitValue(event: UpdateEvent, value: any) {
 			if (
 				(isEqual(value, props.initialValue) ||
 					(props.initialValue === undefined && isEqual(value, defaultValue.value))) &&
 				props.batchMode === false
 			) {
 				emit('unset', props.field);
```

Comment:
Due to the way you are switching events, the first clause in the if-statement will actually cause unexpected problems, as it has value-specific functionality that should only apply when event = updateValue. Right now, you could submit a set-field-value with a value that happens to be the initial value of the current field (like `null`), and promptly call `unset` on the wrong field

Reply:
I didn't see this, my bad ! Fixed in last commit